src/connector.ts

```diff
@@ -70,6 +71,11 @@ export class Connector {
     return data.items;
   }
 
+  public async fetchQuery<T>(params: RequestParams, selector: string): Promise<any> {
+    const { data } = await this.fetchData<Response<T>>(params);
+    return data.data[selector];
```

Comment:
```suggestion
    return data.data[selector];
```
why can't consumer of the function do this `[selector]` unwrap on their own?
you just need to provide correct response type

```
type FooBar = { listApis: TypeOfListApis }

let res = await fetchQuery<FooBar>(...)

work with res.listApis

```